core-v1/src/main/scala/debug/DebugArgs.scala

```diff
@@ -305,6 +306,12 @@ object DebugArgs {
       override type Out = Seq[O]
     }
 
+  implicit def debugMatch[I, S, B, O, OP[_]]: Aux[Expr.Match[I, S, B, O, OP], OP, I, Option[O]] =
+    new DebugArgs[Expr.Match[I, S, B, O, OP], OP] {
+      override type In = I
```

Comment:
I should probably put the matched case here (before the `.thenReturn` expression is applied) as well